scene/3d/node_3d.cpp

```diff
@@ -693,6 +710,8 @@ void Node3D::force_update_transform() {
 }
 
 void Node3D::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("set_basis", "local"), &Node3D::set_basis);
```

Comment:
Why the argument is called `local` instead of `basis`?

Reply:
The existing argument for `set_transform` is also called local. I can change this if desired though.

Reply:
I think changing the argument name should be fine. There is already `set_global_transform` method, so it won't be confusing or something.

Reply:
I actually meant that the argument name in `set_transform()` could be changed too (to `transform`).